requirements/2.7/requirements.txt

```diff
@@ -40,9 +40,6 @@ django-tagging==0.3.1
 pytz==2013.8
 
 # Cthulhu
-M2crypto==0.22.3
```

Comment:
For example: http://ceph.com/calamari/docs/development/dev_env.html#by-hand would be broken by this change, and doing an "install_salt" on normal developer workstations is not necessarily desirable because it's systemwide.

Aside: to avoid all this awkwardness, it would be a good idea to work toward making all this work with upstream package versions in fedora, so that the virtualenv doesn't need a --no-site-packages, and then requirements.txt would only be installing anything that wasn't already installed from fedora RPMs (which would ultimately be nothing).